### PR TITLE
Spring 2018 Fanart Contest results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
tooth and comb plz

### CODE REVIEW

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/banner.jpg)](https://osu.ppy.sh/community/contests/64)
+
+With 77 entries across a variety of illustration styles, Spring 2018 was perhaps the hardest contest we've ever had the joy of judging. 7 votes was nowhere near enough to showcase the sheer mountains of talent that entered the contest this year, but decisions were made, and votes were cast.
+
+Without further ado, here are the finalists from the 2018 Spring Fanart Contest:
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Sunako.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Sunako.jpg)
+
+In first place with a whopping **15923** votes is [Sunako](https://osu.ppy.sh/users/2801166)'s entry! Featuring all of the main four mascots plus Tama enjoying a evening walk throughout a Japanese-themed festival, this wonderfully captivating work is simple, clean, and absolutely adorable.
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: a evening -> an evening

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+It turns out Mocha looks really good in buns, who knew?
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/In%20those%20days.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20In%20those%20days.png)
+
+Trailing super close in second place with **14251** votes is [In those days](https://osu.ppy.sh/users/777173)'s rendition of a spring-attired Pippi talking a walk among sakura-trees in bloom. Simply spectacular, what more is there to say?
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: in those days's -> in those days'
i think

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Earning **7602** votes, [Raynel](https://osu.ppy.sh/users/6318119)'s simple but beautifully stark piece frames both Pippi and Mocha in traditional Japanese attire before a grand estate.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/keizami.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20keizami.jpg)
+
+Putting the selfie in spring, [keizami](https://osu.ppy.sh/users/5402247)'s adorable piece scored **7270** votes, featuring all of the major mascots, and perhaps the first time in a fanart contest where Maria's actually showing her age.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/2Su.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%202Su.png)
+
+Chasing close after with **6734** votes, [2Su](https://osu.ppy.sh/users/6598966)'s amazing work features the major mascots all dressed up in formal springwear, enjoying a walk under a blooming tree. Bonus points from us for one of the nicest Maria cameos we've seen so far as well!
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: springwear -> spring wear

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Sunset overtones herald **4771** votes for [Vio](https://osu.ppy.sh/users/5661237)'s wonderfully unique entry, showcasing Mocha enjoying a ride on her bike and appreciating the local wildlife. A beautifully realised setting that just *oozes* personality. Excellent work!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/R3_AL.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20R3_AL.png)
+
+Achieving **3664** votes, [R3\_AL](https://osu.ppy.sh/users/11983357)'s adorable drawing of a rather aptly titled spring Pippi showcases her wonderfully bright-eyed gaze highlighted with vibrant colouring.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/RaudhrAkai.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20RaudhrAkai.png)
+
+On wings of a spring-tinged dream, **3647** voters were enthralled by [RaudhrAkai](https://osu.ppy.sh/users/1631442)'s fantastical take on an otherwordly Pippi enjoying some springtime scenery. Well done!
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: otherwordly missing an L

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+In third place, [Leissss](https://osu.ppy.sh/users/4750716)'s stunning entry aimed for the stars and scored **13997** votes, set before two torii upon a starlit plain. Amazing use of perspective and colour to really make the surroundings pop just as much as Pippi and Mocha themselves!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Evaine.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Evaine.jpg)
+
+Coming in shortly after with **11675** votes, [Evaine](https://osu.ppy.sh/users/7295733)'s incredible piece, featuring Alisa from the osu!taiko community mascots set to a frankly breathtakingly rendered backdrop using some of the best colouring we've ever seen. Absolutely spectacular, well done!
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Amumizu.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Amumizu.jpg)
+
+Strolling by with **8876** votes total, [Amumizu](https://osu.ppy.sh/users/2920755)'s strolling Mocha features all the natural beauty of spring and our youngest mascot to match.
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: odd to use strolling twice like this, use a synonym

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/Raynel.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20Raynel.png)
+
+Earning **7602** votes, [Raynel](https://osu.ppy.sh/users/6318119)'s simple but beautifully stark piece frames both Pippi and Mocha in traditional Japanese attire before a grand estate.
+
+[![](/wiki/shared/news/2018-05-02-spring-2018-fanart-contest-results/keizami.jpg)](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20-%20keizami.jpg)
+
+Putting the selfie in spring, [keizami](https://osu.ppy.sh/users/5402247)'s adorable piece scored **7270** votes, featuring all of the major mascots, and perhaps the first time in a fanart contest where Maria's actually showing her age.
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: dude is maria ur waifu

File: news/2018-05-02-spring-2018-fanart-contest-results.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+On wings of a spring-tinged dream, **3647** voters were enthralled by [RaudhrAkai](https://osu.ppy.sh/users/1631442)'s fantastical take on an otherwordly Pippi enjoying some springtime scenery. Well done!
+
+---------
+
+Like what you see? [Download all of the entries bundled together in high quality here!](https://assets.ppy.sh/contests/64/winners/osu%21%20Spring%20Fanart%202018%20Winners.zip)
+
+We'd like to thank everyone who entered for helping make the contest so great - this particular fanart contest was accompanied with a LOT of calls for more votes across the board, highlighting the number of incredibly talented entries we got to see.
+
+Each finalist snags themselves 2 months of osu!supporter for themselves or a friend, and the winning pieces are available **NOW** in-game as the Spring seasonal background pack, and will remain live for several weeks. Show the artist of your favourite piece some support and check out their profiles!
```
Comment (Xinnoh) [inline news/2018-05-02-spring-2018-fanart-contest-results.md]: I have no idea what the difference is between the winning pieces and finalists from this post

Comment (Ephemeralis) [reply news/2018-05-02-spring-2018-fanart-contest-results.md]: made this more clear

File: news/2018-05-02-spring-2018-fanart-contest-results.md (hunk 6 above)
Comment (Ephemeralis) [reply news/2018-05-02-spring-2018-fanart-contest-results.md]: yes

**Pull request merged**